src/spokanetech/settings.py

```diff
@@ -19,65 +20,86 @@
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/5.0/howto/deployment/checklist/
 
-# SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = 'django-insecure-t9*!4^fdn*=pmz4%8u_we!88e!8@_!drx0)u_@6$@!nx$4svjp'
+IS_DEVELOPMENT = bool(os.environ.get("SPOKANE_TECH_DEV", False))
+if IS_DEVELOPMENT:
+    # SECURITY WARNING: keep the secret key used in production secret!
+    SECRET_KEY = "django-insecure-t9*!4^fdn*=pmz4%8u_we!88e!8@_!drx0)u_@6$@!nx$4svjp"
 
-# SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+    # SECURITY WARNING: don't run with debug turned on in production!
+    DEBUG = True
 
-ALLOWED_HOSTS = []
+    ALLOWED_HOSTS = []
+else:
+    try:
+        SECRET_KEY = os.environ["DJANGO_SECRET_KEY"]
+    except KeyError as e:
+        raise KeyError(
+            f"{e}: If running in development, set 'SPOKANE_TECH_DEV' to any value."
+        ) from e
+
+    DEBUG = False
+    ALLOWED_HOSTS = ["spokanetech.org"]
+
+    # SSL Options
+    # TODO: These will have to change depending on how infra-platform handles SSL termination
+    # https://docs.djangoproject.com/en/5.0/ref/settings/#secure-hsts-preload
+    SECURE_HSTS_SECONDS = 0
+    SECURE_SSL_REDIRECT = False
```

Comment:
We might want to wait to turn this on until the site is stood up:

> Setting this incorrectly can irreversibly (for some time) break your site. Read the [HTTP Strict Transport Security](https://docs.djangoproject.com/en/5.0/ref/middleware/#http-strict-transport-security) documentation first.
> — https://docs.djangoproject.com/en/5.0/ref/settings/#secure-hsts-seconds

Reply:
Agreed. I set these to the default/off values for now.

Reply:
Oh, duh!